views/job-specification.html

```diff
@@ -0,0 +1,31 @@
+{% extends "layout.html" %} 
+ {% set page_title = "Job Specification" %} {% block main
+    %}
+<main class="container mt-4">
+    <section class="col d-flex justify-content-center mt-4">
+        {% if role %}
+       
+        <div class="card text-center" style="width:75%">
+            <div class="card-body">
+              <h3 class="card-title"> {{ role.jobProfileTitle }}</h3>
+            </div>
+            <ul class="list-group list-group-flush">
+              <li class="list-group-item"> Capability: {{ role.capability }} </li>
```

Comment:
Could you make each of the "headings" bold? For example <strong> Capability:</strong>